test_dir/new_file_3

```diff
@@ -0,0 +1,3 @@
+wow..man
+
+lets see
```

Comment:
RiflerRick committed on a restricted directory: test_dir/new_file_3in the repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir/new_file_3
Collaborators: @manasisingh67, @RiflerRick,